javacutil/src/main/java/org/checkerframework/javacutil/ElementUtils.java

```diff
@@ -215,20 +215,27 @@ public static String getSimpleName(ExecutableElement element) {
      * @param type a type
      * @return the simple type name, without annotations
      */
-    private static String simpleTypeName(TypeMirror type) {
+    public static String simpleTypeName(TypeMirror type) {
```

Comment:
Now that this method is public, we should move it to `TypesUtils` - the parameter type is `TypeMirror`, so that's where I would expect such a method.